ext/lib/encoding/tinycbor/Kconfig

```diff
@@ -0,0 +1,85 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#  http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+#
+
+config TINYCBOR
+	bool
+	prompt "tinyCBOR Support"
+	default n
+	help
+	  This option enables the tinyCBOR library.
+
+if TINYCBOR
+
+config CBOR_NO_DFLT_WRITER
+	bool
+	prompt "No default writer"
+	default n
+	help
+	  This option specifies whether a default writer exists.
+
+config CBOR_NO_DFLT_READER
+	bool
+	prompt "No default reader"
+	default n
+	help
+	  This option specifies whether a default reader exists.
+
+config CBOR_ENCODER_NO_CHECK_USER
+	bool
+	prompt "No check user for cbor encoder"
```

Comment:
Better text: This option controls whether the CborEncoder checks whether the user passed valid arguments.

Reply:
It is a prompt message, hence you cannot put a long descriptive text there.